src/Lykke.Job.CandleHistoryWriter.Repositories/Snapshots/SnapshotCandleEntity.cs

```diff
@@ -39,9 +39,6 @@ public class SnapshotCandleEntity : ICandle
         public DateTime LastUpdateTimestamp { get; set; }
 
         [Key(10)]
```

Comment:
Can we have problems with already saved data? Can old LastTradePrice appears in TradingOppositeVolume?

Reply:
This should not become a problem for we update every service wich is a cosumer of the data as well. The only risk is to update the software prior to trades data migration. The second thing is that here we work around a snapshot serialization/deserialization. It is only needed at shutdown/startup of the job and is rewritten each time.